source/appModules/notepad.py

```diff
@@ -0,0 +1,31 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Joseph Lee
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""App module for Windows Notepad.
+While this app module also covers older Notepad releases,
+this module provides workarounds for Windows 11 Notepad."""
+
+import appModuleHandler
+import api
+
+
+class AppModule(appModuleHandler.AppModule):
+
+	def _get_statusBar(self):
```

Comment:
Could you add a docstring here - here's what comes from super()

```suggestion
	def _get_statusBar(self):
		"""
		Retrieve the status bar object of the application.
		If C{NotImplementedError} is raised, L{api.getStatusBar} will resort to
		a lookup by position.
		"""
```

Reply:
Done - I'll also add a note that status bar child position must be checked whenever Microsoft publishes Notepad updates to stable Windows 11 build users (Notepad 11 is updated through Microsoft Store as opposed to being tied to a Windows release).

Reply:
Can this aspect be included? It's helpful to understand what raising `NotImplementedError` will do in this function.

```
		If C{NotImplementedError} is raised, L{api.getStatusBar} will resort to
		a lookup by position.
```